cores/esp32/esp32-hal-periman.c

```diff
@@ -59,7 +63,7 @@ void * perimanGetPinBus(uint8_t pin, peripheral_bus_type_t type){
 		log_e("Invalid pin: %u", pin);
 		return NULL;
 	}
-	if(type >= ESP32_BUS_TYPE_MAX){
+	if(type >= ESP32_BUS_TYPE_MAX  || type  == ESP32_BUS_TYPE_INIT){
```

Comment:
this is not an invalid call per say. It will always return NULL. No need to print an error really and remember that peripheral manager is internal API

Reply:
This makes the code clear about it. As commented below.

> The added testing for `type == ESP32_BUS_TYPE_INIT` is inteded to make it clear in the code. We shall think that today we are working on it and know that we won't use it with `ESP32_BUS_TYPE_INIT`, but in the future, other people may be working on this code.
> 
> I think that all that makes the code clearer for others, including commentaries, shall be used.